diktat-rules/src/test/kotlin/org/cqfn/diktat/ruleset/smoke/DiktatSmokeTest.kt

```diff
@@ -25,18 +25,14 @@ import org.junit.jupiter.api.BeforeEach
 import org.junit.jupiter.api.Tag
 import org.junit.jupiter.api.Test
 
-import java.io.File
-
 import kotlinx.serialization.encodeToString
 
-typealias ruleToConfig = Map<String, Map<String, String>>
+typealias ruleToConfig = Map<Warnings, Map<String, String>>
```

Comment:
What is it for?

Reply:
To have more consistent arguments in `overrideConfig` method.